packages/itmat-setup/src/databaseSetup/collectionsAndIndexes.ts

```diff
@@ -88,12 +95,19 @@ export async function setupDatabase(mongostr: string, databaseName: string): Pro
     }
 
 
-    /* replace the id from the seeds */
+    /* replace the user id from the seeds */
     seedUsers[0].id = uuid();
     seedUsers[1].id = uuid();
 
     /* insert seed users */
     await db.collection(collections.users_collection.name).insertMany(seedUsers);
 
+    /* replace the organisation id from the seeds */
+    for (const each of Object.keys(seedOrganisations)) {
+        if (seedOrganisations[each].id != 'data-sciene-institute-icl') seedOrganisations[each].id = uuid();
```

Comment:
Why are we checking that things are different from a `data-sciene-institute-icl` ? UUID should be used everywhere.

Reply:
:D @fguitton yeah," data-science-institute-icl" plays as "the single "System" organisation used for the first user created" as you mentioned above :D, other organisations will be "uuid"

Reply:
We still need to get rid of it. This is not a DSI specific software and we need to be as neutral as possible